experimental/sql/tools/rel_opt.py

```diff
@@ -0,0 +1,98 @@
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
This file is supposed to basically mirror the functionality of MlirOpt, so I want to run it directly from the command line with `./tools/rel_opt.py ...`. However, this requires me to have `#!/usrbin/env python3` as the first line. This conflicts with having the license as the first lines. Is there a way to circumvent this? Maybe @nicolasvasilache has an input?